auth-web/tests/unit/components/BcscPanel.spec.ts

```diff
@@ -56,57 +54,31 @@ describe('BcscPanel.vue', () => {
   })
 
   it('doesn\'t render the login or create account link when authenticated', () => {
-    const authenticatedBtns = wrapper.vm.$el.querySelectorAll('.v-btn')
-    const createAccountLink = wrapper.vm.$el.querySelector('.cta-btn')
-
     // Verify only the Learn More Button is rendered
-    expect(authenticatedBtns.length).toStrictEqual(1)
-    expect(authenticatedBtns[0]).toBeDefined()
-    expect(authenticatedBtns[0].textContent).toContain('Learn More')
-
-    // Verify the account create link is not rendered
-    expect(createAccountLink).toBeNull()
+    expect(wrapper.find('.cta-btn').exists()).toBe(false)
+    expect(wrapper.findAll('.v-btn').length).toBe(1)
+    expect(wrapper.find('.learn-more-btn')).toBeDefined()
+    expect(wrapper.find('.learn-more-btn').text()).toContain('Learn More')
   })
 
   it('renders the login button and create account link when NOT authenticated', () => {
     // Render Un-Authenticated
     const wrapper = wrapperFactory({ userProfile: null })
 
     const authenticatedBtns = wrapper.vm.$el.querySelectorAll('.v-btn')
-    const mobileCardLink = wrapper.vm.$el.querySelectorAll('a')
-    const createAccountLink = wrapper.vm.$el.querySelector('.cta-btn')
+    const mobileCardLink = wrapper.findAll('a').at(0)
+    const createAccountLink = wrapper.find('.cta-btn')
 
     expect(authenticatedBtns[0]).toBeDefined()
     expect(authenticatedBtns[0].textContent).toContain('Create a BC Registries Account')
 
-    expect(mobileCardLink[0]).toBeDefined()
-    expect(mobileCardLink[0].textContent).toContain('set up a mobile card')
+    expect(mobileCardLink.exists()).toBe(true)
+    expect(mobileCardLink.text()).toContain('set up a mobile card')
 
-    expect(createAccountLink).toBeDefined()
-    expect(createAccountLink.textContent).toContain('Create a BC Registries Account')
+    expect(createAccountLink.exists()).toBe(true)
+    expect(createAccountLink.text()).toContain('Create a BC Registries Account')
 
     expect(authenticatedBtns[1]).toBeDefined()
     expect(authenticatedBtns[1].textContent).toContain('Learn More')
   })
-
-  it('renders the correct text and number of bullet points', async () => {
```

Comment:
why are you removing this test? Is it no longer valid?

Reply:
ah I see its because the value isn't reactive anymore so the test setup is invalid. Can you just alter it so it still verifies the static bullet points?

Reply:
-- resolving since you have one testing that above